tests/genai/judges/test_make_judge.py

```diff
@@ -366,3 +367,133 @@ def mock_invoke(model_uri, prompt, assessment_name):
+    judge = make_judge(
+        name="test_judge",
+        instructions=original_instructions,
+        model="openai:/gpt-4",
+    )
+
+    assert judge.instructions == original_instructions
+    assert judge.model == "openai:/gpt-4"
+    assert judge.template_variables == {"outputs"}
+    original_description = judge.description
+    assert "Instructions-based judge: test_judge" in original_description
+    assert original_instructions in original_description
+
+    serialized = judge.model_dump()
+    assert "name" in serialized
+    assert serialized["name"] == "test_judge"
+    assert "instructions_judge_pydantic_data" in serialized
+    assert serialized["instructions_judge_pydantic_data"]["instructions"] == original_instructions
+    assert serialized["instructions_judge_pydantic_data"]["model"] == "openai:/gpt-4"
+
+    store = _get_scorer_store()
+    version = store.register_scorer(experiment, judge)
+    assert version == 1
+
+    retrieved_scorer = store.get_scorer(experiment, "test_judge", version)
+    assert retrieved_scorer is not None
+    assert isinstance(retrieved_scorer, InstructionsJudge)
+    assert retrieved_scorer.name == "test_judge"
+    assert retrieved_scorer.instructions == original_instructions
+    assert retrieved_scorer.model == "openai:/gpt-4"
+    assert retrieved_scorer.template_variables == {"outputs"}
+    assert retrieved_scorer.description == original_description
+    assert "Instructions-based judge: test_judge" in retrieved_scorer.description
+    assert original_instructions in retrieved_scorer.description
+
+    deserialized = Scorer.model_validate(serialized)
+    assert isinstance(deserialized, InstructionsJudge)
+    assert deserialized.name == judge.name
+    assert deserialized.instructions == original_instructions
+    assert deserialized.model == judge.model
+    assert deserialized.template_variables == {"outputs"}
+    assert deserialized.description == original_description
+
+    v2_instructions = "Evaluate if the output {{outputs}} is professional, formal, and concise."
+    judge_v2 = make_judge(
+        name="test_judge",
+        instructions=v2_instructions,
+        model="openai:/gpt-4o",
+    )
+    version2 = store.register_scorer(experiment, judge_v2)
+    assert version2 == 2
+
+    versions = store.list_scorer_versions(experiment, "test_judge")
+    assert len(versions) == 2
+
+    v1_scorer, v1_num = versions[0]
+    assert v1_num == 1
+    assert isinstance(v1_scorer, InstructionsJudge)
+    assert v1_scorer.instructions == original_instructions
+    assert v1_scorer.model == "openai:/gpt-4"
+
+    v2_scorer, v2_num = versions[1]
+    assert v2_num == 2
+    assert isinstance(v2_scorer, InstructionsJudge)
+    assert v2_scorer.instructions == v2_instructions
+    assert v2_scorer.model == "openai:/gpt-4o"
+
+    latest = store.get_scorer(experiment, "test_judge")
+    assert isinstance(latest, InstructionsJudge)
```

Comment:
Can we invoke these scorers (`__call__()`) and verify that invoke_judge_model is called with the expected args?